envbuilder.go

```diff
@@ -902,6 +838,340 @@ ENTRYPOINT [%q]`, exePath, exePath, exePath)
 	return nil
 }
 
+// RunCacheProbe performs a 'dry-run' build of the image and checks that
+// all of the resulting layers are present in options.CacheRepo.
+// Logger is the logf to use for all operations.
+// Filesystem is the filesystem to use for all operations.
+// Defaults to the host filesystem.
```

Comment:
Were these arguments previous, but you later moved them to options.Options by any chance?
This comment feels potentially stale.

Reply:
Yep, this is stale 👍